test/integration/resources/payloads/no_changes_payload.json

```diff
@@ -94,6 +82,125 @@
       "OpeningTimeType": "General",
       "AdditionalOpeningDate": "",
       "IsOpen": true
+    },
+    {
+      "Weekday": "",
+      "Times": "10:00-17:00",
+      "OffsetOpeningTime": 0,
+      "OffsetClosingTime": 0,
+      "OpeningTimeType": "Additional",
+      "AdditionalOpeningDate": "Dec 24 2022",
+      "IsOpen": true
+    },
+    {
+      "Weekday": "",
+      "Times": "10:00-17:00",
+      "OffsetOpeningTime": 0,
+      "OffsetClosingTime": 0,
+      "OpeningTimeType": "Additional",
+      "AdditionalOpeningDate": "Dec 25 2022",
+      "IsOpen": true
+    }
+  ],
+  "Contacts": [
+    {
+      "ContactType": "Primary",
+      "ContactAvailabilityType": "Office hours",
+      "ContactMethodType": "Website",
+      "ContactValue": "http://www.boots.com/store-locator#"
+    },
+    {
+      "ContactType": "Primary",
+      "ContactAvailabilityType": "Office hours",
+      "ContactMethodType": "Telephone",
+      "ContactValue": "01288 352906"
+    }
+  ],
+  "Facilities": [
+    {
+      "Id": 1,
+      "Value": "No"
+    },
+    {
+      "Id": 2,
+      "Value": "Yes"
+    },
+    {
+      "Id": 3,
+      "Value": "No"
+    },
+    {
+      "Id": 4,
+      "Value": "Yes"
+    },
+    {
+      "Id": 5,
+      "Value": "No"
+    },
+    {
+      "Id": 6,
+      "Value": "No"
+    },
+    {
+      "Id": 35,
+      "Value": "No"
+    }
+  ],
+  "LastUpdatedDates": {
+    "OpeningTimes": "2021-09-07T10:21:30+00:00",
+    "Facilities": "2021-09-07T10:21:42+00:00",
+    "Services": "2021-09-07T10:21:36+00:00",
+    "ContactDetails": "2017-10-23T14:06:46+00:00"
+  },
+  "OpeningDays": [
```

Comment:
Are we expecting the OpeningDays section from NHSUK?

Reply:
That particular static payload is not in use at this time.. Once the auto change event is introduced the static payloads will be removed